### PR TITLE
Taglines Frontend

### PR DESCRIPTION
**Pull request by nucleogenesis**
### Summary

- New channel layout and cards
- New heading for channel and topic pages

### Reviewer guidance

Review the behavior, layout, and styling of everything in the designs linked below.

I think that having the topic and channel page headings full-width (the text and images) doesn't look good. I think it'd be better at the max-width and centered.

### References

https://github.com/learningequality/kolibri/pull/7058/
https://www.figma.com/file/zbxBoJUUkOynZtgK0wO9KD/Channel-descriptions?node-id=200%3A908
https://docs.google.com/presentation/d/1KQvN4x-oOE5yiS7zKeEBCOeNwTO733V10NL2y8jDsUo/edit#slide=id.g88ad9f844d_0_314

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by jredrejo** [comment]
> * Currently I am showing the description because channels have descriptions... so we'll change that to tagline whenever this PR is ready for review.

@nucleogenesis if you run the migrations from #7058 you can edit your database, and add a `tagline` in your `content_channelmetadata` table . This is an example (using sqlitebrowser in Linux):
image
If you do it, I can see in this PR branch the taglines in the http://localhost:8000/en/device/#/content page (api request http://localhost:8000/device/api/device_channel/?include_fields=on_device_file_size&1592322183348=1592322183348 has this response:
`[{"author":"","description":"Small channel that contains all the Kolibri content types","tagline":null,"id":"95a52b386f2c485cb97dd60901674a98","last_updated":"2020-06-15T13:46:03.639118+02:00","lang_code":"en","lang_name":"English","name":"Kolibri QA Channel","root":"95a52b386f2c485cb97dd60901674a98","thumbnail":"data:image/png;base64,.....8DAOaTBWvv4AAAAAElFTkSuQmCC","version":71,"available":true,"num_coach_contents":8,"public":false,"on_device_file_size":236362416},{"author":"josé","description":"Hello","tagline":"Wow this channel is so amazing","id":"ca7b5a0f3c2c5e32966e41a7e9a04bf3","last_updated":"2020-06-15T19:50:15.778744+02:00","lang_code":"en","lang_name":"English","name":"tagline","root":"ca7b5a0f3c2c5e32966e41a7e9a04bf3","thumbnail":"","version":3,"available":true,"num_coach_contents":0,"public":null,"on_device_file_size":111348}]`
) and in the http://localhost:8000/en/learn/#/topics page where the api request http://localhost:8000/api/content/channel/?available=true&contentCacheKey=1592243483 has a similar response

**Review by jredrejo — COMMENTED** [review]
Added a commit to show the tagline https://github.com/learningequality/kolibri/pull/7061/commits/4bd5deefe18d4c96197df0c34f33322793775be8
image
The green circle in the right channel is a tagline from the database, the red circle is a description because the tagline is None for the left channel.

File: kolibri/plugins/learn/assets/src/views/ChannelCard.vue [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    :to="link"
+    class="card"
+    :style="{ backgroundColor: $themeTokens.surface, color: $themeTokens.text }"
+  >
+    <div
+      class="card-heading"
+      :style="{ borderBottom: `1px solid ${$themeTokens.fineLine}` }"
+    >
+      <h3 class="title" dir="auto">
+        <TextTruncator
```
Comment (jredrejo) [inline kolibri/plugins/learn/assets/src/views/ChannelCard.vue]: I think `TextTruncator` should be used in the card-content for the tagline, not for the title

File: kolibri/plugins/learn/assets/src/views/ChannelCard.vue [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    display: block;
+    width: 100%;
+    height: 48px;
+    padding: $margin;
+    font-size: 12px;
+  }
+
+  /deep/.card-thumbnail-wrapper {
+    max-width: 100%;
+  }
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/views/ChannelCard.vue:217]: this looks potentially brittle 🤔

**Review by jredrejo — APPROVED** [review]
LGTM

File: kolibri/plugins/learn/assets/src/views/ChannelCard.vue (hunk 2 above)
Comment (nucleogenesis) [reply kolibri/plugins/learn/assets/src/views/ChannelCard.vue:217]: The issue is that CardThumbnail sets hard widths and heights so text starts to overlap the image when its KFixedGridItem wrapper shrinks smaller than the fixed height/width img.

I only tested this working properly in Chrome but I'd love some additional testing and eyes on the responsiveness in particular.

**Comment by indirectlylit** [comment]
let's set a max-width on the header content and remove the white background to match the old style:

image

**Review by indirectlylit — APPROVED** [review]
pushed changes

**Comment by indirectlylit** [comment]
force merge for hack session

**Pull request merged**